set-quotas.py

```diff
@@ -202,9 +203,18 @@ def match_keystone_project(all_ks_projects, form_project):
                               quota_list=quota_list,
                               **quota_cfg)
         msg.send()
+        copy_index.append(project['row'])
         print "Successfully updated quotas for project {}".format(
               ks_project.name)
 
+    # FIXME: code is duplicated from addusers.py, centralize it
+    if copy_index:
+        copy_rows = [r for r in rows if rows.index(r) in copy_index]
+        sheet.append_rows(copy_rows, target="Processed Requests")
+        result = sheet.delete_rows(copy_index, 'Form Responses 1')
```

Comment:
`Form Responses 1` is the name of the worksheet/tab in the spreadsheet.  So you pass a list of row numbers, and the name of the worksheet to delete them from.  So here if `copy_index` was `[0, 2, 4]`, it would delete rows 0, 2, and 4 from the worksheet Form Responses 1.